Makefile

```diff
@@ -25,15 +25,13 @@ envinit:
 	go install github.com/vektra/mockery/v2@latest
 	go install github.com/matryer/moq@latest
 	go install golang.org/x/tools/cmd/goimports@latest
-	go install golang.org/x/mobile/cmd/gomobile@latest
-	# The gomobile/gobind libs are also needed when using `gomobile bind`, but it's not compatible with vendoring:
-	# https://github.com/golang/go/issues/50994#issuecomment-1032754206
-	GO111MODULE=off go get -u golang.org/x/mobile/cmd/gomobile
+	# TODO: replace with golang.org/x/mobile/cmd/gomobile@latest once https://github.com/golang/mobile/pull/105 is merged.
+	go install github.com/baizon/mobile/cmd/gomobile@latest
```

Comment:
I forked to https://github.com/BitBoxSwiss/mobile. 

To avoid all the import path renaming stuff, please  `git clone ...` the repo /tmp, then `cd /tmp/gomobile/ && go install .` with a comment that this is a workaround to the module/url mismatch in the forked package.

Reply:
Done.